graylog2-web-interface/src/components/indices/IndexSetsComponent.jsx

```diff
@@ -60,18 +60,26 @@ const IndexSetsComponent = React.createClass({
   },
 
   _formatIndexSet(indexSet) {
+    const menuItems = [];
+
+    if (indexSet.writable) {
+      menuItems.push(<MenuItem onSelect={this._onSetDefault(indexSet)} disabled={!indexSet.writable}>Set as default</MenuItem>);
```

Comment:
I didn't read the issue careful enough and removed the menu item when not writable. I will change it to only disable it in that case.